### ISSUE TITLE
Allow a healthcheck label for docker backendconfigs

### ISSUE THREAD

**Issue by alwaysastudent** [containous/traefik#1286]
### What version of Traefik are you using (`traefik version`)?

```
[root@2ef5302562ab /]# /traefik version
Version:      v1.1.2
Codename:     camembert
Go version:   go1.7.4
Built:        2016-12-15_10:27:40AM
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml...)?
```
   labels:
     - "traefik.backend=search"
     - "traefik.frontend.rule=PathPrefix:/search/v3"
    - "traefik.port=10010"
```

### What did you do?

I am trying to run traefik as a proxy with a docker backend configuration.  We are using docker version 1.11 and it does not support `HEALTHCHECK`. 

### What did you expect to see?

Traefik should be able to hit an url to check the health of the backend instead of relying  on docker's health check alone.  Please support a label like `traefik.backend.health.url` or something similar.

**Comment by timoreimann** [comment]
With 1.2.0, you will already be able to define health checks statically in your provider template file. Look for "healthcheck" in the [docs](https://docs.traefik.io/basics/#backends) to find out how.

Support for defining the health check path dynamically through a label is tracked by #1232. I think that's what you'd ideally like to have, so let's use your ticket to track support in Docker.

**Comment by GabKlein** [comment]
Would be ideal to have this support for docker swarm backend

**Comment by greenmaid** [comment]
+1

**Comment by wenzowski** [comment]
When this is implemented, [will this be the correct usage](https://github.com/wenzowski-docker/traefik/blob/master/docker-compose.yml#L52) for docker?

**Comment by ldez** [comment]
Fixed by #1982

**Comment by wenzowski** [comment]
@timoreimann do you agree with @ldez this is closed by #1982? You noted in #1232 that docker label support was tracked here. If you agree this is closed, does that mean docker labels will not be supported, only dockerfile healthcheck?

**Comment by ldez** [comment]
Sorry I did not see the note in #1232
